lib/pure/strutils.nim

```diff
@@ -1479,7 +1479,7 @@ proc unindent*(s: string): string
     doAssert x == "Hello\nThere\n"
   unindent(s, 1000) # TODO: Passing a 1000 is a bit hackish.
 
-proc delete*(s: var string, first, last: int) {.noSideEffect,
+proc delete*(s: var string, first, last: Natural) {.noSideEffect,
```

Comment:
This is a bad idea, please make it work on ``int``.

Reply:
I decided to copy what `sequtils.delete` already does, since both are the same algorithms and IMO should be consistent.

Reply:
Do not change the signatures, this is a bugfix.